src/jdk.zipfs/share/classes/module-info.java

```diff
@@ -265,7 +268,48 @@
  *       </ul>
  *   </td>
  * </tr>
- *  </tbody>
+ * <tr>
+ *   <th scope="row">accessMode</th>
+ *   <td>{@link java.lang.String}</td>
+ *   <td>null/unset</td>
+ *   <td>
+ *       A value defining the desired read/write access mode of the file system
+ *       (either <em>read-write</em> or <em>read-only</em>).
+ *       <p>
+ *       Even if a zip file system is writable ({@code fs.isReadOnly() == false}),
+ *       this says nothing about whether individual files can be created or
+ *       modified, simply that it might be possible.
+ *       <ul>
+ *           <li>
+ *               If no value is set, the file system is created <em>read-write</em>
+ *               if possible. Use {@link java.nio.file.FileSystem#isReadOnly()
+ *               isReadOnly()} to determine the actual access mode.
+ *           </li>
+ *           <li>
+ *               If the value is {@code "readOnly"}, the file system is created
+ *               <em>read-only</em>, and {@link java.nio.file.FileSystem#isReadOnly()
+ *               isReadOnly()} will always return {@code true}. Creating a
+ *               <em>read-only</em> file system requires the underlying ZIP file to
+ *               already exist, and is incompatible with {@code "create"=true}.
+ *               Specifying both will cause an {@code IllegalArgumentException}
```

Comment:
In the file system API, e.g. Files.newByteChannel or FileChannel.open, the "CREATE" open option is specified to be ignored when opening a file for reading.  The env properties in zipfs doesn't need to match this of course, but they could.